ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/SRMA/Edit/OfferedDate.cshtml

```diff
@@ -0,0 +1,135 @@
+﻿@page "/case/{caseUrn:long}/management/action/srma/{srmaId:long}/offeredDate"
+@model ConcernsCaseWork.Pages.Case.Management.Action.SRMA.EditOfferedDatePageModel
+
+@{
+    ViewData["Title"] = "SRMA Offered Date";
+    ViewData["CancelButtonLabel"] = "Cancel and return to SRMA"; // label used for _Cancel partial view
+    var nonce = HttpContext.GetNonce();
+}
+
+<div class="govuk-width-container">
+    <partial name="_BannerError" />
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            @if (!string.IsNullOrEmpty((string)TempData.Peek("Error.Message")))
+            {
+                <partial name="_Error" />
+            }
+            else if (TempData["SRMA.Message"] != null)
```

Comment:
Can we change this null check to !string.IsNullOrEmpty((string)TempData.Peek("SRMA.Message")